mobile/mysterium/proposals_manager.go

```diff
@@ -109,7 +131,21 @@ func (m *proposalsManager) getProposals(req *GetProposalsRequest) ([]byte, error
 	}
 
 	// Get proposals from remote discovery api and store in cache.
-	apiProposals, err := m.getFromRepository()
+	upperTimePriceBound := uint64(req.UpperTimePriceBound)
```

Comment:
Why not to use uint64 for GetProposalsRequest fields ? No conversion would be needed?